js/data.js

```diff
@@ -1 +1,35 @@
 /* exported data */
+
+function getAdvice() {
+  var dataReq = new XMLHttpRequest();
+  dataReq.open('GET', 'https://api.adviceslip.com/advice');
+  dataReq.responseType = 'text';
+  dataReq.addEventListener('load', function () {
+    JSON.parse(dataReq.response);
```

Comment:
It doesn't look like you are using the response here at all. Is this just for testing?